Directory.Build.props

```diff
@@ -153,7 +153,8 @@
     <_runtimeOS Condition="$(_runtimeOS.StartsWith('tizen'))">linux</_runtimeOS>
     <_runtimeOS Condition="'$(PortableBuild)' == 'true'">$(_portableOS)</_runtimeOS>
 
-    <_packageOS Condition="'$(CrossBuild)' == 'true'">$(_hostOS.ToLowerInvariant)</_packageOS>
+    <_packageLibc Condition="$(_runtimeOS.Contains('musl'))">-musl</_packageLibc>
+    <_packageOS Condition="'$(CrossBuild)' == 'true'">$(_hostOS.ToLowerInvariant)$(_packageLibc)</_packageOS>
```

Comment:
@am11 just to double check, does that change look right to you?

Reply:
@agocke, could you tell what is this fixing?

IMO, we should run official build against the PR as we did previously so we can get some confidence from the package validation step.

ps - what I learned previously about this part of the configuration (`<PropertyGroup Label="CalculateOS">`) is that very tricky to be sure about this part of the configuration. And when everything was green (incl. outerloop and official builds), it broke Tizen, which was reported few days later. 😁

Reply:
Ah, yes, the problem is that PackageRID was `linux-x64` on alpine, when it should be `linux-musl-x64`.